app/src/main/res/values/styles.xml

```diff
@@ -480,6 +480,29 @@
         <item name="android:layout_height">28dp</item>
     </style>
 
+    <style name="ImageView.RoundedCorner" parent="">
```

Comment:
Not sure but the `parent=""` is intended?

Reply:
Removed! (And renamed to reflect its true purpose)